project/publish-docs.sh

```diff
@@ -45,17 +45,35 @@ echo ""
 echo ""
 echo "Commit to gh-pages"
 echo ""
-cd ./gh-pages
+# The curly braces act as a try / catch
+{
+  cd ./gh-pages
+
+  git add ./$1
+  git commit -m "Deploy to GitHub Pages"
 
-# inside this git repo we'll pretend to be a new user
-git config user.name "Travis CI"
-git config user.email "propel@google.com"
+  if [ "$TRAVIS" ]; then
+    # inside this git repo we'll pretend to be a new user
+    git config user.name "Travis CI"
```

Comment:
Doesn't this config need to be set before the commit happens in order to have any effect?